src/Controllers/MovieController.cs

```diff
@@ -77,7 +81,7 @@ public IActionResult EditMovie(MovieViewModel movie, int id)
         }
 
         [HttpGet("")]
-        [HttpGet("ListMovies/{searchString?}")]
+        [HttpGet("ListMovies/{titleSearchString?}/{genreSearchString?}")]
```

Comment:
This parameters should not be in the URL directly (we usually keep that space for IDs).